src/constants/shaders/colorizeUI.frag

```diff
@@ -120,9 +120,9 @@ vec4 getObjectColor(ivec2 sUv, float opacity) {
 
   // color the highlighted object. Note, `highlightedId` is a 0-based index
   // (global ID w/o offset), while `id` is a 1-based index.
-  if (id - 1u == highlightedId) {
-    return vec4(outlineColor, 1.0);
-  }
+  // if (id - 1u == highlightedId) {
+  //   return vec4(outlineColor, 1.0);
+  // }
```

Comment:
Debating whether to just remove this altogether, but `colorizeUI.frag` is mostly adapted from TFE's colorize shader so I thought it might be good to keep them structurally similar?